konfera/event/views.py

```diff
@@ -339,7 +339,7 @@ def render_to_response(self, context):
                                            context=self.get_context_data(), show_content_in_browser=False,
                                            cmd_options={'margin-top': 10, 'zoom': 1, 'viewport-size': '1366 x 513',
                                                         'javascript-delay': 1000, 'footer-center': '[page]/[topage]',
-                                                        'no-stop-slow-scripts': True, 'dpi': 130, 'zoom': 1.33})
```

Comment:
Are you sure the right value is 1?

```
>>> d = {'a': 1, 'b':2, 'a':3}
>>> d['a']
3
```

So I think (not to change the functionality) we should leave the `1.33` and remove the `1`. @ricco386  ?